lib/services/rdbmsManagement/lib/rdbmsManagementClient.d.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+import MySQLManagementClient = require('./mysql/mySQLManagementClient');
+import * as MySQLModels from './mysql/models';
+
+import PostgreSQLManagementClient = require('./postgresql/postgreSQLManagementClient');
+import * as PostgreSQLModels from './postgresql/models';
+
+export { MySQLManagementClient, MySQLModels, PostgreSQLManagementClient, PostgreSQLModels };
```

Comment:
Neat ! I assume this was hand-written? 

Would you please do us a favor - open a terminal, cd to this path and run `tsc.exe` on this `.d.ts` file and ensure it compiles with 0 errors and 0 warnings. We're still working on having a good infrastructure to ensure TS support works, hence the short term requirement to at least execute `tsc.exe` by hand.